### PR TITLE
Update basalt to 0.3.0

### PR DESCRIPTION
**Pull request by jklmnn**

### CODE REVIEW

File: index/ba/basalt/basalt-0.3.0.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description = "Collection of formally verified building blocks"
+name = "basalt"
+version = "0.3.0"
+licenses = "AGPL-3.0-only"
+maintainers = ["kliemann@adacore.com"]
+maintainers-logins = ["jklmnn"]
```
Comment (mosteo) [inline index/ba/basalt/basalt-0.3.0.toml:6]: This would be a good opportunity to add some tags to the release.

Comment (jklmnn) [reply index/ba/basalt/basalt-0.3.0.toml:6]: I added the tags that I found relevant for this library.

**Comment by jklmnn** [comment]
I added a separate version for legacy SPARK versions as the gnatprove currently in alire doesn't seem to support the `Always_Return` annotation.

**Review by mosteo — APPROVED** [review]

**Comment by mosteo** [comment]
Merged, thanks.

**Pull request merged**